frontend/src/main/pages/PlayPage.js

```diff
@@ -80,7 +76,7 @@ export default function PlayPage() {
   // Stryker disable all 
   const mutationbuy = useBackendMutation(
     objectToAxiosParamsBuy,
-    { onSuccess: onSuccessBuy },
+    null,
```

Comment:
Looks like this PR has code from another PR in it... one on which it does not depend.

If these are two independent PRs, we should probably keep the code for them separate.

Reply:
It's PR #16 

It's a good practice to base one PR on a previous one's branch *if and only if* the later PR depends on the code from the earlier branch.

But if the two features are independent, it is better practice to start from the main branch each time.  That allows the code reviews to be cleaner and simpler, and makes it easier to merge.

Reply:
I think if you do this on the current branch, it will fix the issue noted here:

```
git checkout Dav-leaderboard-currency
git pull origin Dav-leaderboard-currency
git pull --rebase origin main
git push origin Dav-leaderboard-currency -f
```

What this does is to rebase your branch on main, which should make the diff clean since these changes have already been merged.

If anything goes wrong with the rebase, you can use this to undo the rebase:
```
git rebase --abort
```

Reply:
Thanks! That's on me for not being clean about my git branches and rushing between issues. Rebased and cleaned.